nilearn/glm/first_level/first_level.py

```diff
@@ -857,6 +862,7 @@ def first_level_from_bids(dataset_path, task_label, space_label=None,
 
     """
     # check arguments
+    sub_labels = sub_labels if sub_labels else []
```

Comment:
You probably want to avoid duplicated values in case users pass twice the same subject by mistake, like the example below:

```python
first_level_from_bids(bids_path, 'main',
                      sub_labels=["01", "01"],
                      space_label='MNI',
                      img_filters=[('desc', 'preproc')])  
```

Reply:
@Remi-Gau maybe we handle this case in a separate PR. I think this is ready to merge as is. WDYT?

Reply:
fine with me.

this is defo good to go for me.